client/src/__tests__/RelatedItems.test.js

```diff
@@ -28,6 +30,20 @@ describe('RelatedProducts', () => {
     const heading = screen.getByText('YOUR OUTFITS');
     expect(heading).toBeInTheDocument();
   });
+
+  // test('should invoke setRelatedIdList function ', async () => {
+  //   const setRelatedIdList = jest.fn();
+  //   jest.spyOn(React, 'useState').mockImplementationOnce((initState) => [initState, setRelatedIdList]);
+  //   const mockAxiosVal = {
+  //     data: [40346, 40350, 40349, 40348],
+  //   };
+  //   axios.get.mockImplementation(() => Promise.resolve(mockAxiosVal));
```

Comment:
maybe try something like axios.get = jest.fn().mockResolvedValueOnce({mockAxiosVal}) to mock the axios request